### ISSUE TITLE
Why not replacing fully connected  operation with  1*1 convolution  in faster-rcnn implementation?

### ISSUE THREAD

**Issue by lyw615** [pytorch/vision#4056]
The   1x1 convolution used in RPN head. But the head of  faster-rcnn  implemented with  nn.Linear(). Why not replacing nn.Linear()  with  1x1 convolution?   The faster-rcnn implementation  refers to torchvision.models.detection.faster_rcnn module.   And i'm also curious about the reason why  conv3x3 for backbone stage  is set to 'bias=False' ?

**Comment by fmassa** [comment]
Hi,

> But the head of faster-rcnn implemented with nn.Linear(). Why not replacing nn.Linear() with 1x1 convolution

Note that replacing the `nn.Linear()` with a 1x1 convolution wouldn't yield the same result. You would need a convolution kernel of size `[resolution, resolution]` size, see https://github.com/pytorch/vision/blob/11480973385719f0eaba6247e65e149867a0b66a/torchvision/models/detection/faster_rcnn.py#L213
But at this point there is really no need of a convolution, and it's just clearer to use a Linear layer instead

> And i'm also curious about the reason why conv3x3 for backbone stage is set to 'bias=False

This is because we have `BatchNorm` after the convolutions, which means that bias in the `conv` layer just before it is not needed.

I believe I've answered your questions, so I'm closing this issue.
